### DISCUSSION TITLE
Telegram, discord ve YouTube puanlarını

### DISCUSSION THREAD

**Discussion by tplayse** [IBAX-io/go-ibax#1519] [Soru Cevap - Türkçe]
Telegram, discord ve YouTube puanlarını  nasıl alacağız?

**Comment by hsyntpl** [comment]
Bende bulamadım.

**Reply by murselbogac0** [reply]
Ben bulamadim

**Comment by Yusufknk** [comment]
Arkadaşlar bunun için yöneticilerle görüşüyoruz. İlk başta form olarak yapacaklardı ama birden fazla bot hesap ihtimali olduğu için bu yöntemi elediler. Başka bir yöntem için çalışıyorlar.

**Reply by MstfYtskn** [reply]
Teşekkürler

**Reply by Zeynel26477575** [reply]
15 saat sonra bu tür puanlar kazanmalar kapatılacak mı? Bu tür konu da bilginiz var mı?

**Comment by murselbogac0** [comment]
1subatta puanlara göre ödüller verilecek diye biliyorum
